src/index.html

```diff
@@ -15,8 +15,99 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="stylesheet"
+      href="https://fonts.google.com/specimen/Roboto"
+    />
```

Comment:
The href attribute for the Google Fonts stylesheet is incorrect. It should be a URL pointing to the font resource, such as 'https://fonts.googleapis.com/css?family=Roboto'.